web/src/styles.css

```
@@ -0,0 +1,877 @@
+  --color-ui-muted-strong: #384238;
+  --color-ui-subtle: #788073;
+
+  --font-body: 400 var(--font-size-body) / var(--line-height-body) var(--font-family-body);
+  --font-caption: 600 var(--font-size-caption) / var(--line-height-tight) var(--font-family-body);
+  --font-caption-strong: 800 var(--font-size-caption) / var(--line-height-tight)
+    var(--font-family-body);
+  --font-chip: 400 var(--font-size-chip) / var(--line-height-body) var(--font-family-mono);
+  --font-control: 600 var(--font-size-control) / var(--line-height-body) var(--font-family-body);
+  --font-editor: 400 var(--font-size-editor) / var(--line-height-editor) var(--font-family-mono);
+  --font-family-body: "Avenir Next", "Segoe UI", "Helvetica Neue", sans-serif;
+  --font-family-display: Georgia, "Times New Roman", serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)